### PR TITLE
Foundations: Update Type Scale

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Latest build: https://e635039.ods.dev/

Latest type test: https://e635039.ods.dev/type-test/

Open questions:

- [x] [How should we best set `html` font-size?](https://github.com/okta/odyssey/pull/903/files#diff-390dbe39690d3602be1b049f6ddee7a5f86060efb90b0dd010dbd495810d2d28R17) (cc @arnoldsandoval-okta)
  - Solving this with Sass math for now; we can move to `calc()` or other once we wrangle custom props.
- [ ] I noticed these variables/maths for Checkbox and Radio, [do we still need these?](https://github.com/okta/odyssey/pull/903/files#diff-afaeeff1aaf3364e6c933a3cf46743201bff860fe36ccd8e1676c2cde72ba1faR19) (cc @arnoldsandoval-okta)
- [x] [How do we want to set the margin for Tables?](https://github.com/okta/odyssey/pull/903/files#diff-a6d9373a9597470bd70423184b561de2bacba025656f0ba46f3e29fd9b955ba0R172) (cc @joejezowski-okta)

### CODE REVIEW

File: packages/odyssey/src/scss/base/_typography-global.scss [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 html {
   font-size: 87.5%;
+  // OQ: Should we do something like "(16/$base-font-size) * 100%"
```
Comment (arnoldsandoval-okta) [inline packages/odyssey/src/scss/base/_typography-global.scss:17]: if the intent is for this to be themeable, I think yes

File: packages/docs/.vuepress/config.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ module.exports = {
       ],
       secondary: [
         { title: 'Updates', link: '/updates/' },
-        { title: 'Roadmap', link: 'https://app.zenhub.com/workspaces/odyssey-5ef0ab09504e9d002794ba2e/roadmap' },
+        { title: 'Roadmap', link: 'https://oktainc.atlassian.net/jira/software/c/projects/OKTA/boards/1104/roadmap' },
```
Comment (edburyenegren-okta) [inline packages/docs/.vuepress/config.js:73]: @jmelberg-okta I believe this is the only file requiring Eng review per CODEOWNERS

**Comment by edburyenegren-okta** [comment]
Sorry, Eng, y'all got pulled in because of our `config.js` change.

@santhoshbalakrishnan this should give you a full scope of the upcoming Type changes.

Visual summary of the changes can be found on the type test page in the description. In short: base size is staying the same, but we are modifying the scale slightly and adjusting title and paragraph margins.

No major or breaking changes for component UI.

**Review by jmelberg-okta — APPROVED** [review]
`*.js` file changes lgtm.

**Review by joejezowski-okta — APPROVED** [review]

File: packages/odyssey/src/scss/components/_checkbox.scss [hunk 3]
```diff
@@ -16,6 +16,8 @@
 
 $size-checkbox: 1em / 0.875;
 
+// OQ: AS, this ^ seems like a variable we could toss / may not be necessary in the new single-baseline world.
```
Comment (arnoldsandoval-okta) [inline packages/odyssey/src/scss/components/_checkbox.scss:19]: Agreed, lets axe it.

**Review by arnoldsandoval-okta — APPROVED** [review]

**Pull request merged**